setup/_update_recipes.rst.inc

```diff
@@ -0,0 +1,38 @@
+3) Updating Recipes
+-------------------
+
+Over time - and especially when you upgrade to a new version of a library - an
+updated version of the :ref:`recipe <recipes-description>` may be available.
+These updates are usually minor - e.g. new comments in a configuration file - but
+it's a good idea to update the core Symfony recipes.
```

Comment:
Actually, contrib recipes as well. What about a minor reword: "it's a good idea to keep your files in sync with the recipes" (or something the like)

Reply:
Indeed, I agree with this change